ocs_ci/ocs/cluster_load.py

```diff
@@ -19,6 +19,22 @@
 logger = logging.getLogger(__name__)
 
 
+def wrap_msg(msg):
+    """
+    Wrap a log message with '=' marks.
+    Necessary for making cluster load background logging distinguishable
+
+    Args:
+        msg (str): The log message to wrap
+
+    Returns:
+        str: The wrapped log message
+
+    """
+    marks = "=" * len(msg)
```

Comment:
I think there shoule be some maximum set as if message will be to long the `=` character will will go to another line and so on, so I think about some maximum number like 120 here or even less like 80? And maybe you don't have to calculate it and just use static one for all cases?

Reply:
yes, good idea, will add this